templates/nginx-conf.j2

```diff
@@ -47,6 +47,12 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+    {% if nginx_proxy_conf_http is defined %}
+      {% for http_conf_line in nginx_proxy_conf_http %}
+{{ http_conf_line }}
```

Comment:
@manics - if I were to close the line with a `;` it stops users defining multi-line config. This is more flexible, allowing e.g. another log format to be defined. Or don't we need/want that extra flexibility?